allennlp/models/semantic_parsing/wikitables/wikitables_decoder_step.py

```diff
@@ -102,17 +128,22 @@ def take_step(self,
         attended_question, attention_weights = self.attend_on_question(hidden_state,
                                                                        encoder_outputs,
                                                                        encoder_output_mask)
+        considered_actions, actions_to_embed, actions_to_link = self._get_actions_to_consider(state)
 
-        # To predict an action, we'll use a concatenation of the hidden state and attention over
-        # the question.  We'll just predict an _embedding_, which we will compare to embedded
-        # representations of all valid actions to get a final output.
-        action_query = torch.cat([hidden_state, attended_question], dim=-1)
-
+        if state.checklist_state is not None:
```

Comment:
Here you'd have to check `if state.checklist_state[0] is not None`.  But that's a really minor complication, compared to all of the places that would be simplified by making it a list of `[None, ...]`.

Reply:
Done.